### DISCUSSION TITLE
how to create dialogue in subtitles?

### DISCUSSION THREAD

**Discussion by faon-92** [SubtitleEdit/subtitleedit#6980] [Q&A]
I see in the setting how to adjust the dialogue style, but I don't know how to create them first.
Just input - before the normal text by hand? 
<img width="191" alt="SubtitleEdit_NGLCviW8AT" src="https://github.com/SubtitleEdit/subtitleedit/assets/127494540/072ec7d5-ea86-48c1-99ff-7770ebbfe198">

**Comment by darnn** [comment]
Yes, it's just plain text, there's nothing complicated about it. Just like:
`Speaker A.`
`-Speaker B.`
or
`-Speaker A.`
`-Speaker B.`
or however you choose to do it.

**Comment by faon-92** [comment]
then the program will recognise these lines as dialogue et apply the dialogue style chose in the settings automatically?

**Comment by darnn** [comment]
I think that's for error correction and such, I'm not 100% sure what instances it's used in. In fact, I just saw that mine was set to the completely wrong thing, and it never seemed to impact me very much one way or the other.

**Comment by faon-92** [comment]
that's what I've found as well, just let it be then, it's ok for me 
thank you Darnn
have a nice day
